turbopack/crates/turbopack-core/src/server_fs.rs

```diff
@@ -58,6 +58,6 @@ impl FileSystem for ServerFileSystem {
 impl ValueToString for ServerFileSystem {
     #[turbo_tasks::function]
     fn to_string(&self) -> Vc<RcStr> {
-        Vc::cell("root of the server".into())
+        Vc::cell("root-of-the-server".into())
```

Comment:
Removed the spaces here to ensure it can be reliably tested. Spaces are never allowed in URLs which mean they get encoded regardless of if you provide them encoded, unlike characters like `[` and `]`